README.md

```diff
@@ -10,7 +10,15 @@ If you discover bugs or deficiencies in the above, please leave an issue to rais
 
 For *new feature proposals*, however, please raise them for [discussion](https://github.com/dotnet/vblang/labels/Discussion), and *only* submit a proposal as a pull request if invited to do so by a member of the Language Design Team (a "champion").
 
-## Discussion
+## Contribute
```

Comment:
I'd like to drop Contribute here and raise "Moving to .NET Core" to a two hashmark level